translations/en.json

```diff
@@ -216,7 +216,7 @@
+    "content": "<p>This is the CESSDA Data Catalogue (CDC) version 3.6.2 released on 2024-07-09. It contains descriptions of the more than 40,000 data collections held by CESSDA’s Service Providers (SPs), originating from over 20 European countries. The CDC is a one-stop shop for searching and discovering European social science data. The presented data includes a range of data types including quantitative, qualitative or mixed-modes data, covering both cross-sectional and longitudinal studies, as well as recently collected and historical data. The metadata (study descriptions) are presented in the language as originally provided by the organisations producing the metadata. Some publishers provide study descriptions both in English and in the local language, some only in either English or in the local language. Currently, about 75%% of study descriptions are available in English.</p><p>The <a href=\"/documentation/\">User Guide</a> presents an overview of how to use the data catalogue for searching, covering basic search operations, application of filters and advanced search functionalities.</p><p>The CDC is a portal for discovering data and detailed dataset descriptions are provided. For information and procedures on how to access data, there is a link [‘Access data’] from each study to the study information on the website of the data provider (Publisher). Refer to the information found there to see which access conditions apply to data created in the particular study.</p><p>There is also an <a href=\"https://www.openarchives.org/pmh/\">OAI-PMH compliant endpoint</a>, so that data aggregators can easily harvest the entire contents of the data catalogue. Record identifiers are unified with those used in the User Interface. It is located at <a href=\"https://datacatalogue.cessda.eu/oai-pmh/v0/oai?verb=Identify\"> https://datacatalogue.cessda.eu/oai-pmh/v0/oai</a>.</p><p>There is also a search API. Documentation is available at the <a href=\"https://api.tech.cessda.eu/\">public REST APIs for CESSDA Services</a> page.</p><h3>Changes since the last version</h3><ul><li>Add support for parsing DDI 3.2 and 3.3 lifecycle documents</li><li>Add related publications from all languages to the study description</li><li>Fixed searchbox input being reset to the start after redirection to the search page</li></ul>"
```

Comment:
The date stays the same? ie. "version 3.6.2 released on 2024-07-09"

Reply:
Good spot. I'll update the date to next Tuesday.